provider/provider.go

```diff
@@ -97,6 +97,13 @@ func StartProvider(flags *pflag.FlagSet) {
 		}
 	}
 
+	if config.ProviderConfig.ProvideAndroid {
+		err = config.SetupGADSWebRTCAndroidApkFile()
+		if err != nil {
+			fmt.Println("There is no GADS Android WebRTC apk uploaded via the Admin UI!!!")
```

Comment:
Consider using a Logger for warnings

Reply:
Fixed in this [commit](https://github.com/shamanec/GADS/pull/176/commits/5ad688daf82f98de49c5743df329b64b4ffa7075)